docs/source/yaml/location_schema.rst

```diff
@@ -12,9 +12,26 @@ The generic location schema, where ``<angle brackets>`` are placeholders, is:
        type: <footprint_type>
        <property>: <footprint_property>
      nav_poses:
-       - [<x1>, <y1>, <z1>, <yaw1>]
-       - ...
-       - [<xN>, <yN>, <zN>, <yawN>]
+       - position:
+           x: <x1>
+           y: <y1>
+         rotation_eul:
+           yaw: <yaw1>
+           angle_units: "degrees"
+       - position:
+           x: <x2>
+           y: <y2>
+         rotation_quat:
+           w: 0.707
+           z: 0.707
```

Comment:
I wonder if here it's also worth using a `<generic>` term like `w: <w2>` etc., just for consistency